app/models/Prepaid.coffee

```diff
@@ -43,6 +44,16 @@ module.exports = class Prepaid extends CocoModel
       
     return 'available'
 
+  typeDescription: ->
```

Comment:
Nitpick and checking If I understand this method correctly. It appears these strings hard coded and not shown to the user? If these are used as keys to indicate a type of prepaid maybe they should either be declared as a constant or a comment to help. I am worried that this string equality checking might break quite easily due to someone accidentally updating the string.

To me this looks like something that should be localized.

Is this shown to the user? A unique key? Is there a finite set of prepaid types? Thank you!

Reply:
@AndrewJakubowicz  thanks for review. the string return by this function is used only by coffee code, not show to the users. and these strings can be only access by call this function, i mean, you cannot break down the logic by changing the string. once the string is changed, for example, change the `'full license'` to `'something strange'` all logic will use `'something stange'` instead. 
the only way to break this logic is make the `if else` returns same string. i think no one will do that.

Reply:
@AndrewJakubowicz oh, wait. user can see these string！ good point. at here when a teacher to select which license to apply:
![image](https://user-images.githubusercontent.com/11417632/93545957-554ff180-f994-11ea-868f-959c8983270d.png)
i capitalize these string now. or will FULL CAPITALIZATION be better?

Reply:
maybe i can set these strings in `locale` with i18n?